dependencies.yaml

```diff
@@ -319,6 +320,14 @@ dependencies:
           - &dask dask>=2022.05.2
           - pytest
           - pytest-cov
+          - boto3>=1.21.21
+      - output_types: [requirements, pyproject]
+        packages:
+          - moto[server]>=4.0.8
+      - output_types: conda
+        packages:
+          - cudf==24.10.*,>=0.0.0a0
```

Comment:
Note that CI is currently failing due to the `cudf` tests running without https://github.com/rapidsai/cudf/pull/16499 being merged yet. The other PR can't be merged until this one is. I think the way to resolve this is to temporarily remove the `cudf` dependency here and below, merge this PR, merge the other one, and then add the `cudf` dependency back.

Reply:
Sounds good

Reply:
I went ahead and commented it out with a note.